package.json

```diff
@@ -11,7 +11,8 @@
     "dev:server": "nodemon --watch 'server/**/*.ts' --exec 'node --loader ts-node/esm --require tsconfig-paths/register' server/index.ts",
     "dev:app": "vite --config app/vite.config.ts",
     "dev": "npm run dev:app & npm run dev:server",
-    "build:app": "tsc app && vite build app",
+    "build:app": "tsc && vite build app",
```

Comment:
This wasn't working previously, I don't think you can run `tsc` in this way:

`npx tsc app
error TS6231: Could not resolve the path 'app' with the extensions: '.ts', '.tsx', '.d.ts', '.cts', '.d.cts', '.mts', '.d.mts'.
  The file is in the program because:
    Root file specified for compilation


Found 1 error.`